tests/libnet/console.go

```diff
@@ -0,0 +1,20 @@
+package libnet
```

Comment:
Why the file is called `console`? `ping.go` also uses the console. Maybe `interface`?

Reply:
Sure, I'll name whatever you want.

Reply:
Done.